packages/affiliates/apps/GetSheetsDevminingStatus.js

```diff
@@ -82,7 +84,8 @@ function parseSheet(sheet, columns = SHEET_COLUMNS) {
       try {
         result[name] = map ? map(row[index]) : row[index];
       } catch (err) {
-        console.error("error in row:" + row, err);
+        // These errors can be distracting, and we really just ignore them for now.
```

Comment:
Are these errors common? What causes them?

Reply:
the spreadsheet can be quite messy, with misformatted fields, missing data or other weird things. If a row errors during parsing we ignore it. This was causing a lot of chatter when evaldo runs automation which i think was confusing for him since the errors could be ignored. 

we just have to make sure on the spreadsheet side that if a row needs to be included, that is formatted correctly, and we can reconcile this with the output from the spreadsheet script.